### DISCUSSION TITLE
Submit transactions and scripts to any network from VSCode

### DISCUSSION THREAD

**Discussion by austinkline** [onflow/flow#1221] [Developer Experience]
Right now I can only submit transactions/scripts in VSCode using emulator accounts. Can we get the ability to submit them to any network supplied in our `flow.json` definitions? Ideally from any account that's assigned to those networks

**Comment by fotescodev** [comment]
@sideninja once we are done with `Flow CLI Supercommands` we could take a look at the VSCode Extension experience end to end

**Comment by devbugging** [comment]
I can see what is the intended UX supposed to look like, and I agree, but we have to first solve the security with accounts, in the current way, if we allow using those accounts in flow.json for mainnet/testnet then this convenience will make it even less secure. After we solve the account security then I think this is a no-brainer and a good improvement/suggestion.
